README.md

```diff
@@ -108,7 +108,7 @@ This is a prefix for Kevin's internal API. You probably don't need to change thi
 -   Type: `Function`
 -   Default: `path => path`
```

Comment:
I think we'll want to update this signature to be `(requestPath, req, res)` instead of just `path`. I'm also noticing that I forgot to update this with my latest change (oops)! Would you mind doing this?
```suggestion
-   Default: `(requestPath, req, res) => requestPath.replace(/^\//, "").replace(/\.js$/, "")`
```